.github/workflows/book.yml

```
@@ -36,7 +36,7 @@ jobs:
     name: Build documentation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@main
+      - uses: actions/checkout@4
```

```suggestion
      - uses: actions/checkout@v4
```
Let's use `v4` everywhere instead of just `4` as it's more descriptive